.github/workflows/integration_test.yaml

```diff
@@ -16,3 +16,16 @@ jobs:
       rockcraft-ref: fastapi-extension
       juju-channel: 3.5/stable
       channel: 1.29-strict/stable
+
+  integration-tests-juju-3-3:
```

Comment:
Should we also run a test on 3.4?

Reply:
The other integration test is on juju 3.5.

Reply:
yeh so with 3.4 we would cover 3.3, 3.4 and 3.5. Would that be an issue? It would make sure we don't miss anything and it is a good boundary test (test on either side and on the boundary)

Reply:
In that case, we can probably update the integration test workflow to a matrix, updated, thanks!